modules/KIWILinuxRC.sh

```diff
@@ -9888,6 +9914,8 @@ function createFilesystem {
         xfs_admin -U $uuid $deviceCreate
     elif [ "$filesystem" = "fat" ];then
         mkfs.fat $opts $deviceCreate $blocks 1>&2
+    elif [ "$filesystem" = "exfat" ];then
+        mkfs.exfat $opts $deviceCreate 1>&2
```

Comment:
mkfs.exfat is required to be part of the initrd, but it's not listed to stay there. See metadata/KIWIConfig.xml
imho we need an entry:

<file name="mkfs.exfat"/>

in the tools section